wire-runtime-swift/src/main/swift/ProtoCodable/ProtoReader.swift

```diff
@@ -305,6 +305,7 @@ public final class ProtoReader {
     }
 
     /** Decode a message field */
+    @_disfavoredOverload
```

Comment:
This coerces the compiler to prefer the proto enum generic over the proto Codable generic

Reply:
neat!